src/core/ext/filters/client_channel/subchannel_stream_client.cc

```diff
@@ -50,7 +50,8 @@ SubchannelStreamClient::SubchannelStreamClient(
       call_allocator_(
           ResourceQuotaFromChannelArgs(connected_subchannel_->args())
               ->memory_quota()
-              ->CreateMemoryAllocator(tracer)),
+              ->CreateMemoryAllocator(GPR_UNLIKELY(tracer_ != nullptr) ? tracer_
```

Comment:
No need for the `GPR_UNLIKELY()` here, since this isn't in a performance-sensitive code path.

If the tracer is null, then I suggest making the string something like "SubchannelStreamClient".

Reply:
These are both reasonable suggestions. I’ll follow up with a force-push.